CHANGELOG.md

```diff
@@ -17,6 +17,7 @@
 * [CHANGE] Ruler: Add `user` and `reason` labels to `cortex_ruler_write_requests_failed_total` and `cortex_ruler_queries_failed_total`; add `user` to
     `cortex_ruler_write_requests_total` and `cortex_ruler_queries_total` metrics. #10536
 * [CHANGE] Querier / Query-frontend: Remove experimental `-querier.promql-experimental-functions-enabled` and `-query-frontend.block-promql-experimental-functions` CLI flags and respective YAML configuration options to enable experimental PromQL functions. Instead access to experimental PromQL functions is always blocked. You can enable them using the per-tenant setting `enabled_promql_experimental_functions`. #10660 #10712
+* [CHANGE] Compactor: Add support for setting per tenant values for `-compactor.max-lookback`. #10794
```

Comment:
This isn't a change, we only note as changes the breaking changes that may affect users, this is an enhancement.

But also, since the feature from #10585 hasn't been included in a Mimir release so far, just add this PR number to that line in the CHANGELOG.